###TASK 7

Write a Python program that reads a number and displays the multiplication table of the given integer.
Sample Input: 15
Sample Output :
15X1=15
15X2=30
...
...
15X10=150

In [33]:
num = int(input(""))

for i in range(1,11):
    print(num, "X", i, "=", (i*num), sep = "") 



15
15X1=15
15X2=30
15X3=45
15X4=60
15X5=75
15X6=90
15X7=105
15X8=120
15X9=135
15X10=150


###TASK 6

Write a Python program that reads a number and finds the sum of the series of 1 +11 + 111 + 1111 +
….+N terms.
=====================================================
Sample Input1:
5
Sample Output1:
1 + 11 + 111 + 1111 + 11111
The Sum is: 1234
=====================================================
Sample Input2:
8
Sample Output2:
1 + 11 + 111 + 1111 + 11111 + 111111 + 1111111 + 11111111
The Sum is: 12345678
====================================================

In [53]:
num = int(input(""))

sum = 0
result = ""
for i in range(1,num+1):
    if i!=num:
        result+= "1"*i + "+"
    else:
        result+= "1"*i
    sum+= int("1"*i)   
print(result)
print(sum)

9
1+11+111+1111+11111+111111+1111111+11111111+111111111
123456789


###5

Write a python function that takes a string with multiple numbers separated by commas as input from
the user. Extract the numbers from the string and make a list and print it. Multiply the numbers of the
list and print the product. [Hint: You can use split()]
=====================================================
Sample Input1:
1,2,3,4,5
Sample output1:
['1', '2', '3', '4', '5']
Product:120
=====================================================
Sample Input2:
10,0,20,3,1
Sample Output2:
['10', '0', '20','3', '1']
Product:0

In [64]:
str1 = input("")
list1 = str1.split(",")
print(list1)
product = 1

for i in list1:
    product*= int(i)
print(product)

10,0,20,3,1
['10', '0', '20', '3', '1']
0


###4

Write a Python program to Capitalize the first character of each word in a String [You cannot use the
built-in upper() function]
Sample Input:
I love python programming
Sample Output:
I Love Python Programming
1

In [1]:
str1 = input("")

result = str1[0]
seen = str1[0]
result1 = str1[0]
for i in str1[1::]:
    if ord(i)== 32:
        if i != seen:
            result += i
            seen = i
        result1+=i    
    else:
        if seen == " ":
            i = (chr(ord(i)-32))
            result += i
            seen = i
        result1+=i     
print(result1) 

I love python programming
I Love Python Programming


###3

Write a Python program that takes two strings from the user. Then remove characters from the first
string which are present in the second string. [you are not allowed to use replace() ]

Sample Input1:
India is great
is
Sample Output1:
nda great
=====================================================
Sample Input2:
this is cSe110 course and we love it
lo1iva
Sample Output2:
ths s cse0 curse nd we e t
=====================================================
Sample Input3:
next course is cse111
lo1iva
Sample Output3:

In [19]:
str1 = input("").lower()
str2 = input("").lower()

for i in str1:
    if i not in str2:
        print(i, end= "")

India is great
is
nda  great

###2

Write a Python program to find the largest and smallest word in a string.
[you are not allowed to use max() and min()]
Sample Input :
It is a string with the smallest and largest word.
Sample Output :
The largest word is “smallest” and the smallest word is 'a'.

In [1]:
str1 = input("")
largest = ""
smallest = ""
index = ""

for i in str1:
    if 65<=ord(i)<=90 or 97<=ord(i)<=122:
        index+= i
    else:
        if len(index)>len(largest):
            largest = index  
        if len(index)<len(smallest) or smallest == "":
            smallest = index
        index = ""    
print(largest)          
print(smallest)    


It is a string with the smallest and largest word.
smallest
a


###1

Write a Python program to count the total number of alphabets, digits, and special characters in a string.
Here, space is considered as a special character.
====================================================
Sample Input:
This is CSE110 Course.
Sample Output:
The number of Alphabets in the string is: 15
The number of Digits in the string is : 3
The number of Special characters in the string is: 4

In [3]:
str1 = input("")

count = 0
count1 = 0
count2 = 0
for i in str1:
    if 65<=ord(i)<=97 or 97<=ord(i)<=122:
        count+=1
    elif 48<=ord(i)<57:
        count1+= 1
    else:
        count2+= 1
        
print(count,count1,count2)        
        

This is CSE110 Course.
15 3 4
